docs/release.md

```diff
@@ -0,0 +1,156 @@
+
+For release candidate:
+
+```
+RELEASE_TYPE=rc make check-chart-kdm-source-values
+```
+
+For final rc and release:
+
+```
+RELEASE_TYPE=final-rc make check-chart-kdm-source-values
+```
+
+There should be no lines showing `INCORRECT`. You can use `make change-final-rc-values` to set the values correctly.
+
+For release candidate:
+
+```
+RELEASE_ACTION=revert make change-final-rc-values
+```
+
+For final rc and release:
+
+```
+RELEASE_ACTION=release make change-final-rc-values
+```
+
+Run `make check-chart-kdm-source-values` with `RELEASE_TYPE` correctly set to check the values again.
+
+## Check open PRs for versions bumps
+
+Pull requests are created automatically when versions of components that Rancher uses are tagged. You can [list open pull requests that have been automatically created](https://github.com/rancher/rancher/pulls?q=is%3Apr+label%3Astatus%2Fauto-created+is%3Aopen), and make sure they are merged. 
+
+## Check for Go module updates
+
+Run `make list-gomod-updates` to list all Rancher Go modules that have updated versions available. Create pull requests to update Go modules using the GitHub Actions workflow [Go Get](https://github.com/rancher/rancher/actions/workflows/go-get.yml)
+
+Example output:
+
+```
+github.com/rancher/apiserver needs update (have: v0.0.0-20220223185512-c4e289f92e46, available: v0.0.0-20220513144301-4808910b5d4d)
+github.com/rancher/dynamiclistener needs update (have: v0.3.1-0.20210616080009-9865ae859c7f, available: v0.3.3)
+github.com/rancher/fleet/pkg/apis needs update (have: v0.0.0-20210918015053-5a141a6b18f0, available: v0.0.0-20220521053957-26e7c98cdb47)
+github.com/rancher/lasso needs update (have: v0.0.0-20220412224715-5f3517291ad4, available: v0.0.0-20220519004610-700f167d8324)
+github.com/rancher/lasso/controller-runtime needs update (have: v0.0.0-20220303220250-a429cb5cb9c9, available: v0.0.0-20220519004610-700f167d8324)
+github.com/rancher/machine needs update (have: v0.15.0-rancher86, available: v0.15.0-rancher9)
+github.com/rancher/norman needs update (have: v0.0.0-20220517230400-5a324b6fc6b1, available: v0.0.0-20220520225714-4cc2f5a97011)
+github.com/rancher/rdns-server needs update (have: v0.0.0-20180802070304-bf662911db6a, available: v0.5.8)
+github.com/rancher/security-scan needs update (have: v0.1.7-0.20200222041501-f7377f127168, available: v0.2.7)
+github.com/rancher/steve needs update (have: v0.0.0-20220415184129-b23977e7f1b5, available: v0.0.0-20220503004032-53511a06ff37)
+github.com/rancher/system-upgrade-controller/pkg/apis needs update (have: v0.0.0-20210727200656-10b094e30007, available: v0.0.0-20220502195742-7eb95a99eb25)
+github.com/rancher/wrangler needs update (have: v0.8.11-0.20220411195911-c2b951ab3480, available: v1.0.0)
+```
+
+## Create tag and push
+
+In this example, the tag is v2.6.6-rc1
+
+```
+cd rancher
+git remote add rancher-release git@github.com:rancher/rancher.git
+git checkout release/v2.6
+git pull rancher-release release/v2.6
```

Comment:
```suggestion
git checkout rancher-release/release/v2.6
```
I would probably recommend checking out the upstream branch directly because someone could have unknown commits in their release branch.